### DISCUSSION TITLE
remark-gfm is extremely slow when transform markdown with a lot gfm content

### DISCUSSION THREAD

**Discussion by chaoyang805** [remarkjs/remark#1125] [Q&A] [answered]
As described in the title, when the markdown content has a lot gfm syntax, such as table. The performance of remark-gfm becomes very poor.
Here is a comparsion between marked and remark-gfm [remark-gfm perf test](https://code.juejin.cn/pen/7208847832260804643). click the run button in the top right corner to see the result.
I'm wondering is there a way to optimize it?

**Comment by wooorm** [comment, marked as answer]
Duplicate of micromark/micromark-extension-gfm-table#6. Please search existing issues and contribute to the discussions that already exist!
